README.md

```diff
@@ -0,0 +1,23 @@
+# Whistler ###
+
+#### Whistler is an error-logging service that generates tickets and allows end-users to report additional details.
+
+
+##### Docker #####
+```
+$ docker build server/
```

Comment:
It fails due to a missing .env file. Will users be responsible for composing that file before running the program?